### PR TITLE
Reorder modal/harmonic and cyclic symmetry examples and add 10-cyclic-symmetry/02-modal_cyclic_no_expansion

### PR DESCRIPTION
**Pull request by cbellot000**

### CODE REVIEW

**Review by PProfizi — APPROVED** [review]
Are there any potentially needed notes about a requirement in DPF Server version, or about ServerContext?

File: examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+# to all cyclic expansion configuration. By default all sectors will be expanded.
+# The cyclic phase (in degree) can be changed with the phi argument.
+u_cyc = dpf.operators.result.cyclic_expanded_displacement(
+    streams_container=model.metadata.streams_provider,
+    sectors_to_expand=[0],
+    time_scoping=[7],
+    phi=0.0,
+)
+# # get the mesh expanded on the first sector for consistency between results and mesh
+mesh_provider = model.metadata.mesh_provider
```
Comment (PProfizi) [inline examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py:64]: The Python class init does not work?

Comment (cbellot000) [reply examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py:64]: I'm not sure to understand the question @PProfizi

File: examples/10-cyclic-symmetry/01-modal_cyclic.py [hunk 2]
```diff
@@ -24,10 +24,11 @@
 # nodes and the first time step. Note that the displacements are expanded using
 # the :func:`read_cyclic
 # <ansys.dpf.core.operators.mesh.mesh_provider.InputsMeshProvider.read_cyclic>`
-# prpoerty with 2 as an argument (1 would ignore the cyclic symmetry).
+# property with 2 as an argument (1 would not perform expansion of the cyclic symmetry).
```
Comment (JennaPaikowsky) [inline examples/10-cyclic-symmetry/01-modal_cyclic.py:27]:
```suggestion
# property with 2 as an argument (1 does not perform expansion of the cyclic symmetry).
```

File: examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
+# noqa: D400
+"""
+.. _ref_basic_cyclic:
+
+Get base and duplicate sectors (real and imaginary) result for modal cyclic symmetry
```
Comment (JennaPaikowsky) [inline examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py:5]:
```suggestion
Get base and duplicate sectors (real and imaginary) results for modal cyclic symmetry
```

File: examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+###############################################################################
+# Create the model and display the state of the result.
+model = dpf.Model(examples.find_simple_cyclic())
+print(model)
+
+###############################################################################
+# Get base and duplicate sectors displacement results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# By default, the result providers (stress, displacement...) will return results for
```
Comment (JennaPaikowsky) [inline examples/10-cyclic-symmetry/02-modal_cyclic_no_expansion.py:23]:
```suggestion
# By default, the result providers (stress, displacement, and so on) will return results for
```

**Pull request merged**